web/src/dialog_widget.ts

```diff
@@ -130,6 +130,8 @@ export function get_current_values($inputs: JQuery): Record<string, unknown> {
                 current_values[property_name] = this.files[0];
             } else if (property_name === "edit_bot_owner") {
                 current_values[property_name] = $(this).find(".dropdown_widget_value").text();
+            } else if ($(this).hasClass("pill-container")) {
+                current_values[property_name] = $(this).find(".pill-value").text();
```

Comment:
So a pill container can have multiple pills; how would this logic work correctly in that case?

Reply:
The `$(this).find(".pill-value")` selector will locate all elements with the class `.pill-value`, and applying `.text()` to this selector will concatenate the text values of each `.pill-value` element into a single string.

Reply:
OK but why is it correct to concatenate them; wouldn't we want an array their metadata?

Reply:
OK the answer is it isn't really, but since we're just trying to check if anything has changed, it basically works.